module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -196,7 +200,7 @@ public function createSession(
      *
      * @return array    The requested record
      */
-    public function retrieve(
+    public function retrieveEdsItem(
```

Comment:
There are tests that rely on the existence of a retrieve method. We either need to fix the tests or modify our approach to differentiated retrieval. I think for backward compatibility reasons, it might be preferable to keep retrieve in some form, even if only as a deprecated wrapper.

Reply:
Ok, I added it back as a pass-through and with a deprecation warning.

Reply:
Sounds good. If we're considering retrieve to be deprecated, we should also update the test that calls it (though that's obviously not the most urgent task at hand here :-) ).

Reply:
I updated the unit test as well, since otherwise it was failing (since now I have to mock retrieveEdsItem, not retrieve).